app/uk/gov/hmrc/apinotificationqueue/controller/QueueController.scala

```diff
@@ -45,6 +46,7 @@ class QueueController @Inject()(queueService: QueueService, fieldsService: ApiSu
   def save(): Action[AnyContent] = Action.async {
     implicit request => {
       val headers = request.headers
+      Logger.debug(s"[QueueController] Request received. Headers=$headers")
```

Comment:
Do you need to log all headers? or just a specific one?
Let's NOT add noise if not needed.

Reply:
I think you also need to ensure these NEVER get logged in ET or PROD unless you can guarantee there is no PII in any of those headers.  Go GDPR ;-)